src/App.js

```diff
@@ -109,4 +110,4 @@ const AppHolder = () => {
   )
 }
 
-export default AppHolder
+export default hot(AppHolder)
```

Comment:
why we're wrapping here always but routers are wrapped web only ?

Reply:
wrapping should only happen for web. fixed.